.github/workflows/deploy.yml

```diff
@@ -1,9 +1,10 @@
-name: Deploy
+name: deploy
 
 on:
   push:
-    branches:
-      [master, google_analytics]
+    branches: [master, google_analytics]
+  workflow_dispatch:
```

Comment:
why do we need a workflow dispatch event here?

Reply:
Oh! this was already setup before the handover. Need to check once